.github/workflows/check-and-manage-shavar-files-updates.yml

```diff
@@ -0,0 +1,116 @@
+name: Run script on shavar-creation-lists
+on:
+  schedule:
+    # Weekly on Monday at 00
+    - cron: '0 0 * * 1'
+
+permissions:
+  contents: 'read'
+  id-token: 'write'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false 
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - uses: actions/checkout@v3
+      with:
+        repository: 'irios3/shavar-list-creation'
+        ref: 'creat-json-tracking-protection-lists'
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: '${{ secrets.GCLOUD_AUTH }}'
+        export_environment_variables: true
+    - name: Download version file
+      run: |
+        echo "Download version.txt file"
+        curl -X GET -o "version.txt" https://storage.googleapis.com/shavar-lists-ios-public/Public/version.txt
+    - name: Locally generate shavar JSON files
+      run: |
+        ls
+        mv sample_shavar_list_creation.ini shavar_list_creation.ini
+
+        echo "Run script to get JSON files"
+        python3 lists2safebrowsing.py
+
+        echo "Lists are generated"
+        ls
+
+        # Once the files are generated, check which is the latest version of these files
```

Comment:
After syncing the branch with latest main, I was able to get all versioned files (including 108). Previously we could not get files from 98 on and the approach we took to compare files and if there were differences create a version, was wrong. Having all the versions generated simplies this script.